src/Raven.Server/Documents/TimeSeries/TimeSeriesPolicyRunner.cs

```diff
@@ -176,6 +208,23 @@ internal async Task HandleChanges()
 
                         await AddNewPolicy(collectionName, prev, policy.Policy);
                     }
+
+                    //TODO: clean up the removed policies from the rollups table
```

Comment:
Aren't they going to be cleaned up automatically when we read them, see they are not relevant and removed it?

Reply:
right, done.